packages/playwright-base/env/index.js

```diff
@@ -25,11 +25,58 @@ module.exports = composeEnv([require("@kie-tools/root-env/env")], {
       default: "false",
       description: "Toggles the installation of Playwright dependencies. Can be `true` or `false`.",
     },
+    PLAYWRIGHT_BASE__enableGoogleChromeTestsForAppleSilicon: {
+      default: "true",
+      description: "Enable Google Chrome tests for ARM OSs. Overrides PLAYWRIGHT_BASE__enableGoogleChromeProject.",
+    },
+    PLAYWRIGHT_BASE__enableChromiumProject: {
+      default: "true",
+      description: "Enable or disable Chromium project",
+    },
+    PLAYWRIGHT_BASE__enableGoogleChromeProject: {
+      default: "true",
+      description: "Enable or disable Google Chrome project",
+    },
+    PLAYWRIGHT_BASE__enableWebkitProject: {
+      default: "true",
+      description: "Enable or disable Webkit project",
+    },
+    PLAYWRIGHT_BASE__projectTimeout: {
+      default: "90000",
+      description: "The timeout of a test in the project (Chromium, Google Chrome and Webkit)",
+    },
+    PLAYWRIGHT_BASE__expectTimeout: {
+      default: "30000",
+      description: "The timeout of an expect assertion",
+    },
+    PLAYWRIGHT_BASE__maxDiffPixelRatio: {
+      default: "0.001",
+      description: "The threshold of the maximum difference on screenshot comparisons (in percentage)",
```

Comment:
Is `0.001` = 0.001% or 0.1%? If it's 0.1% I would use `(percentage in decimal form)` just to be a bit clearer.

```suggestion
      default: "0.001",
      description: "The threshold of the maximum pixel difference on screenshot comparisons (percentage in decimal form)",
```